src/flexipages/NPSP_Get_Started.flexipage

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FlexiPage xmlns="http://soap.sforce.com/2006/04/metadata">
+    <flexiPageRegions>
+        <componentInstances>
+            <componentName>gsVideoHeader</componentName>
+        </componentInstances>
+        <componentInstances>
+            <componentName>gsAdminSetup</componentName>
+        </componentInstances>
+        <name>region1</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentName>gsApplicationStatus</componentName>
+        </componentInstances>
+        <name>region2</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentName>gsResources</componentName>
+        </componentInstances>
+        <name>region3</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <masterLabel>Get Started</masterLabel>
```

Comment:
@smarovich Do you think the new tab should be labeled "NPSP Get Started" or just "Get Started"? There could be other products installed in the org that have similar "Get/Getting Started" tabs, so I think having NPSP in the tab name helps make it clear which product this applies to.

Reply:
I am going to defer to Ali and Anne on this one. I'll check in with them and let you know as soon as I hear back.

Reply:
I chatted with Ali and she suggested "Get Started with NPSP". Would that fit?

Reply:
I like that a lot. Thanks!
FYI @dcaballeroSf